packages/macos-at-driver-server/MAINTAINING.md

```diff
@@ -20,9 +20,11 @@ locally (that is, without publishing it for use by others).
 ### Instructions
 
 1. Build the Xcode project
+   1. Select `Open Existing Project...` from Xcode and navigate to `/packages/macos-at-driver-server/MacOSATDriverServer/MacOSATDriverServer.xcodeproj` in your local repository
+   2. With the project open, select `Product>Build` from the top-level menu
 2. Execute the following command in a terminal:
 
-       $ npm pack
+   $ npm pack
```

Comment:
The indentation here is intentional: it signifies a code block within a list item. [Without it, the command renders a plain text](https://github.com/bocoup/at-driver-servers/blob/75f4d5cd26a626318203d4e1391b3e88f6f893cc/packages/macos-at-driver-server/MAINTAINING.md)--not the end of the world but not an obvious improvement, either. Would you be cool with leaving this part as-is?

Reply:
Yup! I think my editor may have stripped that... will revert.